config/schema/README.md

```diff
@@ -0,0 +1,105 @@
+# Schema definitions
+
+This directory tree holds the schema definitions for our search indexes.
+The purpose of the schema is to define:
+
+ - which fields are allowed to be present in documents passed to the search
+   indexer
+ - how these fields should be processed both at search indexing time
```

Comment:
This doesn't read quite correctly...  is 'both' left over from a previous wording of this?